subprojects/android-test/snackbar-rule/src/main/kotlin/com/avito/android/test/app/second/SnackbarChecks.kt

```diff
@@ -0,0 +1,63 @@
+package com.avito.android.test.app.second
+
+import com.avito.android.test.waitFor
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.MatcherAssert
+import org.hamcrest.TypeSafeMatcher
+import ru.avito.util.Is
+
+class SnackbarChecks internal constructor(
+    private val proxy: SnackbarRule.Proxy
+) {
+
+    fun isShownWithTextMatches(text: Matcher<String>) {
```

Comment:
I'd prefer to see in API just overloaded methods when we have different matchers.

```kotlin
isShownWithText("text")
isShownWithText(contains("text"))
```